components/Footer.tsx

```diff
@@ -1,18 +1,120 @@
-const Footer = () => {
+import {
+  Slack,
+  Facebook,
+  Instagram,
+  Github,
+  Mail,
+  ExternalLink,
+  Bug,
+} from "lucide-react";
+import Link from "next/link";
+import Image from "next/image";
+import { useTheme } from "../lib/hooks/useTheme";
+
+const footerLinks = [
+  { name: "Slack", icon: <Slack />, link: "https://onlinentnu.slack.com/" },
+  {
+    name: "Facebook",
+    icon: <Facebook />,
+    link: "http://facebook.com/LinjeforeningenOnline",
+  },
+  {
+    name: "Instagram",
+    icon: <Instagram />,
+    link: "https://www.instagram.com/online_ntnu/",
+  },
+  { name: "Github", icon: <Github />, link: "https://github.com/appKom" },
+];
+
+export default function Footer() {
+  const theme = useTheme();
+
   return (
-    <div className="w-full py-5 px-5 flex flex-col items-center gap-3 text-online-darkTeal dark:text-white">
-      <div>
-        Skjedd en feil? Ta kontakt med{" "}
-        <a
-          className="font-semibold underline transition-all hover:text-online-orange"
-          href="mailto:appkom@online.ntnu.no"
-        >
-          Appkom
-        </a>{" "}
-        :)
+    <footer className="px-4 py-12 mt-24 text-gray-700 bg-white border-t border-gray-300 dark:text-gray-300 dark:border-gray-700 md:px-6 lg:px-8 dark:bg-gray-800">
```

Comment:
Foreslår litt mørkere bakgrunn for å skille det tydligere fra hovedsiden i frontend.

```suggestion
    <footer className="px-4 py-12 mt-24 text-gray-700 bg-zinc-100 border-t border-gray-300 dark:text-gray-300 dark:border-gray-700 md:px-6 lg:px-8 dark:bg-gray-800">
```